Editor/AGS.Editor/Panes/FreePanControl.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace AGS.Editor
+{
+    public class FreePanControl : BufferedPanel
+    {
+        private const int WS_EX_COMPOSITED = 0x02000000;
+        private int _lastX, _lastY; //used to prevent useless mouse move events
```

Comment:
Use a `Point` instead, then you can also change the code in `OnMouseMouse` to `e.Location == lastLocation` and `lastLocation = e.Location`.